changelog/deprecate_enforceValidFormatSpec.dd

```diff
@@ -0,0 +1,4 @@
+Deprecate std.format : enforceValidFormatSpec
+
+`enforceValidFormatSpec` from `std.format` has been accidentally made
+public and will be removed from public view in 2.107.0.
```

Comment:
Is there an alternative to using `enforceValidFormatSpec`? Probably nobody is using it, but just in case.

Reply:
No, I don't think so - as I wrote above: "I did not manage to find a useful workaround here, but IMHO there is no real use case either."